src/components/breadcrumbs/breadcrumbs.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import { Breadcrumbs } from "./breadcrumbs";
+import { nodesTrail } from "./breadcrumbs.mock";
+
+describe("Parent links", () => {
+  const breadcrumbAssumptions = [
+    { title: "Home", href: "/" },
+    { title: "News", href: "/news/" },
+    { title: "Podcasts", href: "/news/podcasts/" },
+    {
+      title: "Why the world needs more women leaders in science and business",
+      href: "/news/podcasts/women-leaders/",
+    },
+  ];
+
+  it("it should link to parent pages", () => {
+    renderBreadcrumbs();
+
+    const links = screen.getAllByRole("link");
+    expect(links).toHaveLength(breadcrumbAssumptions.length);
+    links.map((link, index) => {
+      expect(link).toHaveAccessibleName(breadcrumbAssumptions[index].title);
+      expect(link).toHaveAttribute("href", breadcrumbAssumptions[index].href);
+    });
+  });
+});
+
+function renderBreadcrumbs() {
+  render(
+    <MemoryRouter>
+      <Breadcrumbs nodes={nodesTrail} />
+    </MemoryRouter>
+  );
+}
```

Comment:
Could we pass props to this render function - I like that we don't have to repeat the boilerplate around the component, but ideally once we've added more tests with different props (e.g. Testing correct cases, Testing incorrect cases / Error handling, Testing extreme cases) we'll want to invoke the same boilerplate but with different props being passed through.

Something like
```tsx
const defaultProps: BreadcrumbsProps = {
  nodes: nodesTrail,
};

function renderBreadcrumbs(props?: BreadcrumbsProps = defaultProps) {
  render(
    <MemoryRouter>
      <Breadcrumbs {...props} />
    </MemoryRouter>
  );
}
```